frame/message-queue/src/tests.rs

```diff
@@ -593,7 +596,7 @@ fn bump_service_head_bails() {
 
 #[test]
 fn bump_service_head_trivial_works() {
-	new_test_ext::<Test>().execute_with(|| {
```

Comment:
You can also fix the tests by inserting proper books into the state.  
Otherwise for these small tests it is also okay to remove the assertion since it is expected to not work.